base/linalg/schur.jl

```diff
@@ -28,15 +39,55 @@ function getindex(F::Schur, d::Symbol)
     end
 end
 
+doc"""
+
+    schur(A::StridedMatrix) -> T::Matrix, Z::Matrix, λ::Vector
+
+Computes the Schur factorization of the matrix `A`. The methods returns the (quasi) triangular Schur factor `T` and the orthogonal/unitary Schur vectors `Z` such that `A = Z*T*Z'`. The eigenvalues of `A` are returned in the vector `λ`.
```

Comment:
redundant plural in "The methods returns"

please add line breaks, painful to review without them

remove the leading empty line in the docstrings